cmd/commands/cli/command.go

```diff
@@ -21,29 +21,41 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"path/filepath"
 	"strconv"
 	"strings"
 
 	"github.com/chzyer/readline"
+	"github.com/mysteriumnetwork/node/cmd"
 	"github.com/mysteriumnetwork/node/metadata"
 	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/mysteriumnetwork/node/tequilapi/endpoints"
 	"github.com/mysteriumnetwork/node/utils"
+	"github.com/urfave/cli"
 )
 
+const cliCommandName = "cli"
+
 // NewCommand constructs CLI based with possibility to control quiting
-func NewCommand(
-	historyFile string,
-	tequilapi *tequilapi_client.Client,
-) *Command {
-	return &Command{
-		historyFile: historyFile,
-		tequilapi:   tequilapi,
+func NewCommand() *cli.Command {
+	return &cli.Command{
+		Name:  cliCommandName,
+		Usage: "Starts a CLI client for Tequilapi",
+		Action: func(ctx *cli.Context) error {
+			nodeOptions := cmd.ParseFlagsNode(ctx)
```

Comment:
What if daemon already running in the system? It will conflict with an existing service and it's undefined which one user want's to connect.

Reply:
But `service` is already working like that. All command should stay consistent.
Currently it's simple - new daemon stops with error, because port 4050 is taken.

Reply:
Added starting a node.

Reply:
Understood now. We should do something like `startOrUseExisting`